### PR TITLE
sonar report processed output file schema changes

### PR DESCRIPTION
**Pull request by pkuma-msft**
Minor changes in the sonar processed report schema. Earlier version was outputting an array of 'issue' objects. Newer version outputs a JSON object which has a 'status' key and 'issues' key. In earlier scenario, the processed report wasn't getting uploaded as build artifact if there were no new issues, in the current code the file gets uploaded as below: 
{
    "status":  "Analysis Complete",
    "issues":  [
               ]
}

This is useful in VSO to identify that the analysis was successful but no new issues were found.

### CODE REVIEW

**Comment by jsharmaMSFT** [comment]
Looks good to me.

**Comment by duncanpMS** [comment]
LGTM

File: Tasks/SonarQubePostTest/CodeAnalysisFilePathComputation.ps1 [hunk 1]
```diff
@@ -153,16 +153,19 @@ function ProcessSonarCodeAnalysisReport
 
     $sonarReportFilePath = GetSonarReportFilePath $agentBuildDirectory
     $sonarReportProcessedFilePath = GetSonarReportProcessedFilePath $agentBuildDirectory
+
```
Comment (bgavrilMS) [inline Tasks/SonarQubePostTest/CodeAnalysisFilePathComputation.ps1]: You could create the object lower down, where you add more properties to it

Comment (pkuma-msft) [inline Tasks/SonarQubePostTest/CodeAnalysisFilePathComputation.ps1]: Yup, will take care of this in next iteration..

**Comment by bgavrilMS** [comment]
LGTM

**Pull request merged**